### ISSUE TITLE
Broken link to MegaDepth dataset links

### ISSUE THREAD

**Issue by SunghwanHong** [lixx2938/MegaDepth#50]
Hi, 
I'm trying to download MegaDepth  SfM Models, but it seems that all the links in the project page are broken.
https://www.cs.cornell.edu/projects/megadepth/

Can you please check the link? 

Thank you

**Comment by leon5678** [comment]
hi , I have the same problem, did you download the 667GB dataset successfully?

**Comment by SunghwanHong** [comment]
@leon5678 yeah, the link seemed to be back online when I tried again. But I'm not sure if it works now as well. Maybe the link connectivity is unstable
